src/status_im2/contexts/chat/lightbox/zoomable_image/style.cljs

```diff
@@ -6,14 +6,16 @@
 (defn container
   [{:keys [width height]}
    {:keys [pan-x pan-y pinch-x pinch-y scale]}
+   full-screen-scale
```

Comment:
consider making this function use 1 map rather than a combination of maps and params.